assets/chef-recipes/cookbooks/mdbe_build/recipes/default.rb

```diff
@@ -86,40 +84,68 @@
 ]
 
 debian_packages = %w[
-  default-jdk
-  python-dev
   unixodbc
 ]
 
 debian_stretch_packages = %w[
+  default-jdk
   autoconf
   automake
   dh-systemd
   libgnutls28-dev
   libgnutls30
   libjemalloc1
+  dpatch
+  netcat
+  python-dev
+  libboost-filesystem-dev
 ]
 
 debian_buster_packages = %w[
+  default-jdk
   dh-systemd
   libjemalloc2
   libpmem-dev
+  dpatch
+  netcat
+  python-dev
+  libboost-filesystem-dev
 ]
 
+
 debian_bullseye_packages = %w[
+  default-jdk
   debhelper
   libjemalloc2
   libpmem-dev
   liburing-dev
+  dpatch
+  netcat
+  python-dev
+  libboost-filesystem-dev
 ]
 
+debian_bookworm_packages = %w[
+  debhelper
+  libjemalloc2
+  libpmem-dev
+  liburing-dev
+  netcat-traditional
+  python-dev-is-python3
+  default-jdk
+]
+#  libboost-filesystem-dev - NOT found
```

Comment:
Works fine for me. Tried on both GCP boxes. This package is also included in `debian_and_ubuntu_packages`  list. So eliminating it from bookworm makes no effect.